src/features/device-state/routes/state-v3.ts

```diff
@@ -0,0 +1,412 @@
+import type { RequestHandler } from 'express';
+import type { Request } from 'express';
+
+import * as _ from 'lodash';
+import {
+	captureException,
+	handleHttpErrors,
+} from '../../../infra/error-handling';
+import {
+	ConfigurationVarsToLabels,
+	filterDeviceConfig,
+	formatImageLocation,
+	getReleaseForDevice,
+	readTransaction,
+	rejectUiConfig,
+	serviceInstallFromImage,
+	setMinPollInterval,
+	varListInsert,
+} from '../utils';
+import { sbvrUtils, errors } from '@balena/pinejs';
+import { events } from '..';
+import { Expand } from 'pinejs-client-core';
+
+const { UnauthorizedError } = errors;
+const { api } = sbvrUtils;
+
+type LocalStateApp = StateV3[string]['apps'][string];
+type ServiceComposition = AnyObject;
+export type StateV3 = {
+	[uuid: string]: {
+		name: string;
+		is_managed_by__device?: string;
```

Comment:
```suggestion
		is_managed_by_device?: string;
```

Is there a reason some fields correspond directly to the model and others are renamed specifically for the target state?

Reply:
Which ones don't correspond? The only one that comes to mind is `release_uuid`/`commit` and that's because `commit` in the model should become `release_uuid` to better represent what it actually is but that work just hasn't been done yet